lbrynet/extras/daemon/Daemon.py

```diff
@@ -2598,8 +2598,8 @@ def jsonrpc_channel_list(self, account_id=None, page=None, page_size=None):
         elif old_txo.claim.is_signed and not clear_channel and not replace:
             channel = old_txo.channel
 
-        if 'fee_address' in kwargs:
-            self.valid_address_or_error(kwargs['fee_address'])
+        if {'fee_currency', 'fee_amount', 'fee_address'}.intersection(kwargs.keys()):
```

Comment:
You don't need `.keys()`, this works just as well: `{'fee_currency', 'fee_amount', 'fee_address'}.intersection(kwargs)`